src/utils/markdown-parser/get-nested-text.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Find the text node in a nested array
+ * @param {Array} child an ast children array
+ * @returns {String} description text
+ */
+export const getNestedText = (child: any): string => {
+    if (!child) {
```

Comment:
Wondering if this could be written like so:

```
if (child) {
        const firstChild = child[0];
        // If this child has child arrays keep digging
        if (firstChild.children) {
            return getNestedText(firstChild.children);
        }
        // If we hit the text node we can return that value
        if (firstChild.type === 'text') {
            return firstChild.value;
        }
    }
    
    // There wasn't any content
    return '';
```